ui/lib/core/addon/helpers/cluster-states.js

```diff
@@ -0,0 +1,69 @@
+import { helper as buildHelper } from '@ember/component/helper';
+
+// A hash of cluster states to ensure that the status menu and replication dashboards
+// display states and glyphs consistently
+// this includes states for the primary vault cluster and the connection_state
+
+export const CLUSTER_STATES = {
+  running: {
+    glyph: 'check-circle-outline',
+    isOk: true,
+    isSyncing: false,
+  },
+  ready: {
+    glyph: 'check-circle-outline',
+    isOk: true,
+    isSyncing: false,
+  },
+  'stream-wals': {
+    glyph: 'check-circle-outline',
```

Comment:
i changed the glyph here to `check-circle-outline` instead of `android-sync` based off of conversations with bk that `stream-wals` is in fact the ideal syncing state.